### ISSUE TITLE
Mac Office — Bold and Bold Italic style linking not working for some fonts

### ISSUE THREAD

**Issue by RosaWagner** [MicrosoftDocs/typography-issues#829]
It is quite recent, I noted inconsistent behaviours with the Bold and Italic buttons. 
- For example Baskerville, Bold button shows Bold Italic, few days the Bold+Italic was showing a faux bold italic.
- I tried with imported fonts that I tested in the past, the variable version works but the static version is also show a faux bold-italic when using the buttons (they are not named the same)

It does look pretty random though but driving me crazy at the moment.

I tried on M1 Big Sur, and Intel Mojave. I cleaned caches, at some point I also uninstalled/re-installed. I also tested with the beta version. Works fine on Windows 10.

**Comment by RosaWagner** [comment]
I post this issue here cause there might be a bad interpretation of the opentype spec, or maybe the spec have changed regarding naming scheme and OS/2 parameters, who knows. If it's bug from the softwares, and if you know where to report these bugs please advise :)

**Comment by kenmcd** [comment]
It appears Apple has sabotaged the style groups in a number of Apple-supplied fonts.
So you have to be careful when testing Bold and Italic buttons using any of their fonts.

This same issue came up in the Affinity software forums.
So I have been testing many of the Apple-supplied font families.
And in many of the font families, the style groups are broken.
As this has been the same over many OS versions, and over a long time period - and it has never been fixed - one can only assume it has been done on purpose.
Or ... Apple font engineers had a collective brain fart years ago and have not recovered.

Baskerville is one of those font families in which the style groups are broken.
If you extract the Baskerville.ttc file and check the individual fonts, 
you will see that all six fonts are in the same style group.
Since this includes the Regular, SemiBold, and the Bold weights ...
there are two style group Regular, and two style group Italic - in one style group. 

Many other font families are broken - in many different ways.
In some families the way the family is broken is quite random, like someone just picked a field and thought "let's change this one this time".

Some of the TTC files will not open in the Windows font viewer.
A comment in the FontCreator forum said this is because there is an invalid checksum in the file - more evidence of sabotage.
FontCreator will open the TTCs and extract the individual TTF files, but those TTF files will also be invalid and will not open in the Windows font viewer.
The files have to be saved again to get valid checksums.
After fixing the names and re-saving, the fonts work fine in Windows font viewer.

This may also have implications when embedding a font in an MS Office file (like in your other post) - is an invalid font file being embedded?
Or is just wrong/broken name info being saved into the file?

So when testing Bold and Italic buttons in an app that uses style groups on a Mac,
be sure the Apple-supplied fonts are not broken.

**Comment by RosaWagner** [comment]
Thanks for your reply.

I am also testing fonts from Google Fonts and Fonts from private clients. `Bold Italic` style linking is broken (in Mac not Windows) for most of them (OTF, TTF, Variable). For the open source ones, you can look at JetBrains Mono, Red Hat fonts Text and Display, or Texturina for example. Cormorant (GF) works… I couldn't find a relevant difference that could explain that. 

We didn't change our build system in the last months and I tested these fonts before — it was working at that time. So I don't think the fonts are broken. I would say either there is a bug in the software, or a specification has changed that invalidates perfectly fine fonts for Mac Office. 

> you will see that all six fonts are in the same style group.
Since this includes the Regular, SemiBold, and the Bold weights ...
there are two style group Regular, and two style group Italic - in one style group.

What do you mean by style group? Which name ID are referring to?

**Comment by tiroj** [comment]
Style group = name ID 1, i.e. the RIBBI 4-style family.

Rosalie, do the fonts in question have the same name ID 1 and 2 settings for both Mac and Windows platforms? I learned recently that Glyphs 2 was automating different settings for these on the mistaken belief that the platform settings in the name table were what different OS’s always used. [The context for that discovery was the Macintosh platform settings being used in Powerpoint on Windows.]

**Comment by jvgaultney** [comment]
Both we and some of our font users noticed that the breaking of Bold Italic seems to have happened in macOS Word v16.52, as 16.51 did not exhibit the problem (we had tested that), and we noticed the change the moment we upgraded to 16.52. It happened to me and others without any intervening macOS updates. Our fonts have only platform 3 (Windows) IDs and are consistent. See https://github.com/silnrsi/font-gentium/issues/6

**Comment by RosaWagner** [comment]
@tiroj OK thanks, I didn't know if he was referring to NameID 1 (family name) or 2 (style name). 

> Rosalie, do the fonts in question have the same name ID 1 and 2 settings for both Mac and Windows platforms?

Yes of course ;) 

>Both we and some of our font users noticed that the breaking of Bold Italic seems to have happened in macOS Word v16.52, as 16.51 did not exhibit the problem

@jvgaultney brilliant, many thanks, it leads towards the direction of a bug from the software with the last update.

**Comment by RosaWagner** [comment]
@jvgaultney I copy-paste here the microsoft issue ticket you linked in the gentium issue, in case other people passing by want to add up to the report:
https://techcommunity.microsoft.com/t5/office-for-mac/bold-italics-not-working-on-word/m-p/2625182

**Comment by jvgaultney** [comment]
What's also odd is that the bug affects bold italic in documents, however the font menu shows it correctly:

![image](https://user-images.githubusercontent.com/4226750/135463088-1521aa5f-8d55-4756-b547-703291d02ffb.png)

**Comment by RosaWagner** [comment]
It seems to fixed in 16.55 ! 😄

**Comment by jvgaultney** [comment]
No, it's not fixed in 16.55, as I reported 5 days ago. Here's a screenshot proof: 

![image](https://user-images.githubusercontent.com/4226750/135996962-302ec109-2472-41fc-a720-db65dad49cfc.png)

**Comment by jvgaultney** [comment]
I also have another user that confirms that 16.55 still has problems.

**Comment by jvgaultney** [comment]
I have now updated to 16.55 **build 21100201** and it does now seem to be fixed! I'll now try to further confirm this with users.

**Comment by RosaWagner** [comment]
yes sorry I didn't detailed the build number
